app/test/fuzz_abci_test.go

```diff
@@ -30,26 +30,64 @@ func TestPrepareProposalConsistency(t *testing.T) {
 		accounts[i] = tmrand.Str(20)
 	}
 
-	testApp, kr := util.SetupTestAppWithGenesisValSet(accounts...)
-
 	type test struct {
 		name                   string
 		count, blobCount, size int
+		iterations             int
 	}
 	tests := []test{
-		{"many small single share single blob transactions", 1000, 1, 400},
-		{"one hundred normal sized single blob transactions", 100, 1, 400000},
-		{"many single share multi-blob transactions", 1000, 100, 400},
-		{"one hundred normal sized multi-blob transactions", 100, 4, 400000},
+		// running these tests more than once in CI will sometimes timout, so we
+		// have to run them each once per square size. However, we can run these
+		// more locally by increasing the iterations.
+		{"many small single share single blob transactions", 1000, 1, 400, 1},
+		{"one hundred normal sized single blob transactions", 100, 1, 400000, 1},
+		{"many single share multi-blob transactions", 1000, 100, 400, 1},
+		{"one hundred normal sized multi-blob transactions", 100, 4, 400000, 1},
```

Comment:
note to self, we should try to make these tests faster. Generating all the txs or something is taking forever, so we're only running each of these once per size